provider/datagen/src/source.rs

```diff
@@ -61,24 +58,27 @@ impl SourceData {
     /// The latest segmentation LSTM model tag that has been verified to work with this version of `icu_datagen`.
     pub const LATEST_TESTED_SEGMENTER_LSTM_TAG: &'static str = "v0.1.0";
 
-    #[doc(hidden)]
+    /// Creates a [`SourceData`] that downloads data from the latest tested tag.
```

Comment:
Can you leave this as-is? `Default` should be the canonical API

Reply:
The user is making a choice about the provenance of their data and `SourceData::latest_tested()` is 1000% more clear than `Default::default()`.